[INFO   ] Logger: Record log in /storage/emulated/0/Project/unse/.kivy/logs/kivy_21-03-22_0.txt
[WARNING] [Config      ] Upgrading configuration in progress.
[WARNING] [Config      ] Older configuration version detected (0 instead of 21)
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.8.3 (default, Jun 15 2020, 02:19:10) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 v1.r19p0-01rel0.###other-sha0123456789ABCDEF0###'>
[INFO   ] GL: OpenGL vendor <b'ARM'>
[INFO   ] GL: OpenGL renderer <b'Mali-G71'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <8192>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] KivyMD: v0.104.1
[INFO   ] Text: Provider: sdl2
[INFO   ] GL: NPOT texture support is available
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 696, in _apply_rule
[WARNING] stderr:     setattr(widget_set, key, value)
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 35, in kivy.weakproxy.WeakProxy.__setattr__
[WARNING] stderr:   File "kivy/properties.pyx", line 497, in kivy.properties.Property.__set__
[WARNING] stderr:   File "kivy/properties.pyx", line 544, in kivy.properties.Property.set
[WARNING] stderr:   File "kivy/properties.pyx", line 599, in kivy.properties.Property.dispatch
[WARNING] stderr:   File "kivy/_event.pyx", line 1214, in kivy._event.EventObservers.dispatch
[WARNING] stderr:   File "kivy/_event.pyx", line 1096, in kivy._event.EventObservers._dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/label.py", line 354, in _trigger_texture_update
[WARNING] stderr:     self._create_label()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/label.py", line 347, in _create_label
[WARNING] stderr:     self._label = CoreMarkupLabel(**dkw)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/text/markup.py", line 94, in __init__
[WARNING] stderr:     super(MarkupLabel, self).__init__(*largs, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/text/__init__.py", line 288, in __init__
[WARNING] stderr:     self.resolve_font_name()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/text/__init__.py", line 356, in resolve_font_name
[WARNING] stderr:     raise IOError('Label: File %r not found' % fontname)
[WARNING] stderr: OSError: Label: File 'basic.ttf' not found
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Project/unse/main.py", line 143, in <module>
[WARNING] stderr:     Main().run()
[WARNING] stderr:   File "/storage/emulated/0/Project/unse/main.py", line 69, in __init__
[WARNING] stderr:     input_ui = InfoScreen("infoscreen")
[WARNING] stderr:   File "/storage/emulated/0/Project/unse/main.py", line 60, in __init__
[WARNING] stderr:     super().__init__()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/relativelayout.py", line 265, in __init__
[WARNING] stderr:     super(RelativeLayout, self).__init__(**kw)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/floatlayout.py", line 65, in __init__
[WARNING] stderr:     super(FloatLayout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/layout.py", line 76, in __init__
[WARNING] stderr:     super(Layout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/widget.py", line 359, in __init__
[WARNING] stderr:     self.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/widget.py", line 467, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 536, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 705, in _apply_rule
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "<inline>", line 40:
[WARNING] stderr: ...
[WARNING] stderr:      38:                text: ""
[WARNING] stderr:      39:                font_name: "basic.ttf"
[WARNING] stderr: >>   40:                markup: True
[WARNING] stderr:      41:                multiline: True
[WARNING] stderr:      42:                size_hint: 0.9, 0.9
[WARNING] stderr: ...
[WARNING] stderr: OSError: Label: File 'basic.ttf' not found
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 696, in _apply_rule
[WARNING] stderr:     setattr(widget_set, key, value)
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 35, in kivy.weakproxy.WeakProxy.__setattr__
[WARNING] stderr:   File "kivy/properties.pyx", line 497, in kivy.properties.Property.__set__
[WARNING] stderr:   File "kivy/properties.pyx", line 544, in kivy.properties.Property.set
[WARNING] stderr:   File "kivy/properties.pyx", line 599, in kivy.properties.Property.dispatch
[WARNING] stderr:   File "kivy/_event.pyx", line 1214, in kivy._event.EventObservers.dispatch
[WARNING] stderr:   File "kivy/_event.pyx", line 1096, in kivy._event.EventObservers._dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/label.py", line 354, in _trigger_texture_update
[WARNING] stderr:     self._create_label()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/label.py", line 347, in _create_label
[WARNING] stderr:     self._label = CoreMarkupLabel(**dkw)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/text/markup.py", line 94, in __init__
[WARNING] stderr:     super(MarkupLabel, self).__init__(*largs, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/text/__init__.py", line 288, in __init__
[WARNING] stderr:     self.resolve_font_name()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/text/__init__.py", line 356, in resolve_font_name
[WARNING] stderr:     raise IOError('Label: File %r not found' % fontname)
[WARNING] stderr: 
